packages/azos-ui/vcl/util/sticky-container.js

```diff
@@ -21,36 +21,33 @@ export class StickyContainer extends Control {
   #isFixed = false;
   #startingOffset = 0;
 
-  #scrollListener = null;
-  #resizeListener = null;
+  #scrollListener = () => {
```

Comment:
My only question is about the style.
I think we discussed this before, the "lambda" syntax.

```
 #scrollListener = () => { ...}
vs 
 #scrollListener(){ ...} 


```

Reply:
The lambda syntax has a bit different "this" handling, but I think in this case it does not matter?

Matt, can you please test it both ways, I would prefer to have jut private methods instead of lambda for consistence, as "lambda" syntax adds some extra concerns